File: other_language/c#/lec2_main_p2.cs
using System;

namespace OOP_Lec2
{
    class Object1
    {
      static void Main()
      {
        
        Employee e1 = new Employee();
        e1.SetID(1);
        e1.SetName("Ahmed");
        e1.SetSalary(1000);

        Console.WriteLine(e1.GetID());
        Console.WriteLine(e1.GetName());
        Console.WriteLine(e1.GetSalary());


        // after adding proparity
        Employee e2 = new Employee();
        e2.id = 10;
        e2.NAME = "Mostafa";
        e2.SALARY = 5000;

        Console.WriteLine(e2.id);
        Console.WriteLine(e2.NAME);
        Console.WriteLine(e2.SALARY);



        
      }
    }
    
}

File: other_language/java/lec2_main_p2.java
// package oop_lec2;
// public class OOP_Lec2
public class Main
{
	public static void main(String[] args)
    {
        
        Employee e1 = new Employee();
        e1.SetID(1);
        e1.SetName("Ahmed");
        e1.SetSalary(1000);
        
        System.out.println(e1.GetID());
        System.out.println(e1.GetName());
        System.out.println(e1.GetSalary());
    }
    
}
